try {
    throw TypeError("Types are wrong");
} catch TypeError as e {
    print("Caught");  # expect: Caught
    print(e.message());  # expect: Types are wrong
    print(e);  # expect: TypeError: Types are wrong
               # expect: Traceback (most recent call last):
               # expect:   File "test/generic_builtins/type_error.gen", line 2, in <script>
               # expect:         throw TypeError("Types are wrong");
               # expect:         ~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
}

throw TypeError("Wrong again!");  # expect runtime error: TypeError: Wrong again!
